factcast-factus/src/main/java/org/factcast/factus/Factus.java

```diff
@@ -220,4 +221,24 @@ default <P extends SubscribedProjection> void waitFor(
    */
   @NonNull
   FactStore store();
+
+  /**
+   * @return Current time in seconds since 1970-01-01 00:00:00-00 (can be negative) from the
+   *     factstore.
+   *     <p>Can be used to synchronize across clients (e.g. to decide if it is too late to publish a
+   *     certain event)
+   */
+  default long currentTimeInMillis() {
```

Comment:
Think we can drop this one. the extra effort of wrapping into Instant is insignificant.